scripts/tests/Ice/adapterDeactivation.py

```diff
@@ -3,7 +3,10 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
 
-from Util import TestSuite
+from Util import CSharpMapping, Darwin, TestSuite, platform, Mapping
 
+# Disable IPv6 for macOS until https://github.com/dotnet/runtime/pull/108334 is merged in .NET 8
+# See https://github.com/zeroc-ice/ice/issues/2061
+ipv6 = not (isinstance(Mapping.getByPath(__name__), CSharpMapping) and isinstance(platform, Darwin))
 
-TestSuite(__name__, multihost=False)
+TestSuite(__name__, multihost=False, options={"ipv6": [ipv6]})
```

Comment:
When we set ipv6 to true, what happens? Do we still run the test with ipv4?

Reply:
Here IPv6 means the tests supports ipv6, this affects the combinations that are run when you use `--all`. With this the macOS run with `--all` would not include `--ipv6` in the generated combinations.